commons/commons-serialization/src/test/kotlin/ac/uk/ebi/biostd/tsv/TsvDeserializerTest.kt

```diff
@@ -153,7 +153,10 @@ class TsvDeserializerTest {
                     Attribute("Grant Id", "No. 2015BAD27B01")),
                 sections = mutableListOf(
```

Comment:
we may add a test and handle case (or todo with ticket) when parent does exist.

Reply:
This case is already handled by the TSV serializer. I added a unit test for this case.